benchmark/src/main/java/com/aws/greengrass/clientdevices/auth/benchmark/AuthorizationBenchmarks.java

```diff
@@ -37,7 +37,7 @@
 
 
 @BenchmarkMode(Mode.AverageTime)
-@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@OutputTimeUnit(TimeUnit.SECONDS)
```

Comment:
this will still give seconds per operation right? Rather than ops per second

Reply:
oh right, read it too quickly, so throughput then? yep will adjust